pkg/fanal/analyzer/language/ruby/gemspec/gemspec.go

```diff
@@ -31,6 +32,10 @@ func (a gemspecLibraryAnalyzer) Analyze(_ context.Context, input analyzer.Analys
 		return nil, xerrors.Errorf("%s parse error: %w", input.FilePath, err)
 	}
 
+	for i, lib := range libs {
+		libs[i].License = licensing.Normalize(lib.License)
+	}
+
```

Comment:
I don't think we need it as it is done in `ToAnalysisResult`.
https://github.com/aquasecurity/trivy/blob/846ca243902e351f97aa07955f9f4dd568276b2b/pkg/fanal/analyzer/language/analyze.go#L36-L39